region/crateria/central.json

```diff
@@ -3135,6 +3174,47 @@
                 }
               ],
               "devNote": "No strat for getting through the Boyons with a dboost, since it's redundant with the bluesuit jump."
+            },
+            {
+              "id": 5,
+              "strats": [
+                {
+                  "name": "Boyons Cleared",
+                  "notable": false,
+                  "requires": [{"obstaclesCleared": ["A"]}]
+                },
+                {
+                  "name": "Take Boyon Damage",
+                  "notable": false,
+                  "requires": [
+                    {"enemyDamage": {"enemy": "Boyon","hits": 1,"type": "contact"}}
+                  ]
+                },
+                {
+                  "name": "Ice",
+                  "notable": false,
+                  "requires": [],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": ["Ice"]
+                    }
+                  ]
```

Comment:
Might be worth mentioning that this won't kill the Boyons, but it removes them as an obstacle

Reply:
2 -> 5 and reverse might have a screw attack strat that doesn't use the obstacle.
I'm thinking screw attack and one of: careful jump, speed booster, space jump